package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cozy-notes",
-  "version": "1.17.0",
+  "version": "1.17.1",
```

Comment:
No need to upgrade! The latest released version is still on 1.16.0. So we still continue to populate the 1.17.0 release for now.

Reply:
Yes the 1.17 is not yet released you're right.